### DISCUSSION TITLE
Citation at start of footnote generates a secondary footnote: intended behaviour?

### DISCUSSION THREAD

**Discussion by lyndondrake** [jgm/pandoc#9885] [Q&A]
Hi,

If I have a footnote that is like this:

```
[^7]: @Eruera1997. Eruera's text switches freely between English and
    Māori, and in this instance I have translated the Māori part of the
    sentence.
```

Pandoc generates a secondary footnote, which looks like this in Word:

<img width="594" alt="Screenshot 2024-06-17 at 9 01 53 PM" src="https://github.com/jgm/pandoc/assets/18509357/1077b294-48f4-4338-9bcf-a005052b9838">

It is missing both the citation for `@Eruera1997` and creates the non-intended footnote '1' inside the intended footnote '14'. A similar effect happens in `LaTeX`, so I don't think it's an artefact of the Word writer. If I add text prior to `@Eruera1997` I get an inline citation, which is what I was expecting, so it seems to be specific to the first element of a footnote being a citation.

Is this intended behaviour, and if so is there anything I can do to force an inline citation?

**Comment by jgm** [comment]
What pandoc version?
Assuming you've tried on the latest version, can you give us everything needed to reproduce the issue?

**Comment by lyndondrake** [comment]
Apologies for my very slow response. I have attached a minimal file to demonstrate the problem. Note that it refers to a CSL file which I have also included (from the citation-styles repository; I had to add a `.txt` extension to get the file to upload which you should remove). The PDF output from running:

`pandoc --citeproc -s -o pandoc-footnote-issue.pdf pandoc-footnote-issue.md`

is also attached. From messing around with having `--citeproc` in the command line or not, and doing `.md` output, it looks to my unskilled eye as if something is happening to turn the `@Eruera1997` into a footnote or enumeration before citeproc even tries to recognise the inline citation as a citation?


[pandoc-footnote-issue.md](https://github.com/user-attachments/files/16730688/pandoc-footnote-issue.md)
[society-of-biblical-literature-fullnote-bibliography.csl.txt](https://github.com/user-attachments/files/16730697/society-of-biblical-literature-fullnote-bibliography.csl.txt)
[pandoc-footnote-issue.pdf](https://github.com/user-attachments/files/16730703/pandoc-footnote-issue.pdf)

**Comment by jgm** [comment]
OK, I see the issue.  Your footnote is:
```
[^1]: @Eruera1997. This is a footnote.
```
The contents of a footnote are parsed like other block-level content.  But if you just had
```
@Eruera1997. This is a footnote.
```
It would be parsed as an [example list](https://pandoc.org/MANUAL.html#numbered-example-lists).  So that is what is happening.

You can defeat this by escaping the period
```
@Eruera1997\.
```
or turning off the `example_lists` extension.

I'm wondering whether there's something we could do to avoid this, as this seems likely to come up frequently.  There's an ambiguity that must be resolved between example lists and citations, and you might think pandoc could do that by just recognizing that `Eurera1997` is in the bibliography.  However, the parser doesn't necessarily have access to the bibliography, so it can't be done this way. For now, the escaped period is the best I can recommend.

**Reply by lyndondrake** [reply]
That's helpful. The other thing that seems to disambiguate is to use curly braces, e.g. `@{Eruera1997}`, which I can probably apply a bit more routinely, I think.

In terms of recognising example lists, is there a reason why the parser doesn't require parentheses around the @Eruera1997, i.e. `(@Eruera1997)` as in the examples given in the documentation?

**Reply by jgm** [reply]
An example list can have any of the three delimiter styles: (@foo), @foo), or @foo.
